models/appointment.js

```diff
@@ -37,7 +37,9 @@ const AppointmentSchema = mongoose.Schema({
 { timestamp: true})
 
 AppointmentSchema.pre('save', async function() {
-    this.bookDate = new Date()
+    const defaultDate = new Date();
+    defaultDate.setHours(0,0,0,0)
```

Comment:
what does this do?

Reply:
it sets the bookdate for an appointment to 12am of the current day.